client/app.tsx

```diff
@@ -8,10 +8,21 @@ import { UserContextProvider, useUser } from './context/UserContext.js';
 import Header from './components/Header/Header.js';
 import Footer from './components/Footer/Footer.js';
 import Muncher from './components/Muncher/Muncher';
+import { createSpeechlySpeechRecognition } from '@speechly/speech-recognition-polyfill';
+import SpeechRecognition from 'react-speech-recognition';
 
 const container = document.getElementById('app') || document.createElement('div');
 container.id = 'app';
 const root = createRoot(container);
+
+if (process.env.REACT_APP_SPEECHLY_API == null) {
```

Comment:
I'm curious about the structure of this if statement. What motivated you to check specifically if the value is null? If it's not specified in your env won't it come back undefined? Or why not use:
```js
if (!process.env.REACT_APP_SPEECHLY_API)
```
to check for all falsy values?

Reply:
it is a guard against a possible null value, typescript gets angry otherwise